src/main/java/com/google/sps/servlets/AddEventServlet.java

```diff
@@ -40,10 +40,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     // Only Moderators can add Events.
-    if (user.isModeratorOfAnyOrg()) {
-      // TODO: Can Maintainers add Events?
-      throw new IllegalArgumentException("ERROR: User does not have the permission to perform addition of Events.");
-    }
+    // if (user.isModeratorOfAnyOrg()) {
```

Comment:
would we not want to keep this? in the case that a user submits a form they would have to be a maintainer or moderator. the original if statement is incorrect though i think

Reply:
this is also just commented out for the sake of developing / testing the frontend-backend connection

Reply:
It actually should be !user.isModeratorOfAnyOrg(). My latest change reflects this!

Reply:
Can you add the change jnnybk was suggesting?